frodoProtocols.m

```diff
@@ -406,6 +417,23 @@
 end
 
 
+%%
+function [pulseCommandArray] = currentStepPulse( pulseAmp, pulseDur )
+ephysSettings;
+
+BREAK_DURATION = 0.05; %sec
+
+preStepCommand = zeros(1, BREAK_DURATION * rigSettings.sampRate );
```

Comment:
Could rename this to preAndPostStepCommand  or spacerStepCommand or something, NBD